core/db.py

```
@@ -294,3 +297,120 @@ async def init_db_async() -> None:
 
     async with _ASYNC_ENGINE.begin() as conn:
         await conn.run_sync(metadata.create_all)
+
+
+def get_db_connection():
+    """Get a database connection for testing purposes.
+
+    RU: Возвращает соединение с базой данных для тестирования.
```

Ruff (0.13.3)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)